src/main/frontend/handler/editor.cljs

```diff
@@ -1910,14 +1947,23 @@
 
 (defn handle-last-input []
   (let [input           (state/get-input)
+        input-id        (state/get-edit-input-id)
+        edit-block      (state/get-edit-block)
         pos             (cursor/pos input)
-        last-input-char (util/nth-safe (.-value input) (dec pos))
-        last-prev-input-char (util/nth-safe (.-value input) (dec (dec pos)))
-        prev-prev-input-char (util/nth-safe (.-value input) (- pos 3))]
+        content         (.-value input)
+        last-input-char (util/nth-safe content (dec pos))
+        last-prev-input-char (util/nth-safe content (dec (dec pos)))
+        prev-prev-input-char (util/nth-safe content (- pos 3))]
 
     ;; TODO: is it cross-browser compatible?
     ;; (not= (gobj/get native-e "inputType") "insertFromPaste")
     (cond
+      (and (= content "1. ") (= last-input-char " ") input-id edit-block
```

Comment:
@xyhp915 Now that block lists are the default, can we disable the dwim lists by default? This would just be a one-line change since it's already configurable in config.edn e.g. `:dwim/settings { :list? false }`. The dwim list ux is kinda buggy - https://www.loom.com/share/7843e5759e4647edae6e8704c7c1990f

Reply:
I disabled dwim lists as I think ❤️ means you're in agreement. Feel free to revert if not

Reply:
@logseq-cldwalker I can't seem to enable to old list DWIM. Changing the toggle value has no effect.